src/OMSimulatorLib/Model.cpp

```diff
@@ -632,8 +646,15 @@ void oms::Model::exportSignalFilter(pugi::xml_node &node) const
   {
     pugi::xml_node oms_variable = node.append_child(oms::ssp::Version1_0::oms_Variable);
     oms_variable.append_attribute("name") = signal.c_str();
-    //oms_variable.append_attribute("type") = getTypeString(connector->getType()).c_str();
-    //oms_variable.append_attribute("kind") = getCausalityString(connector->getCausality()).c_str();
+
+    oms::ComRef cref(signal);
+    cref.pop_front();
```

Comment:
The signal names within the signal filter shouldn't start with the model name. So this should be considered as a workaround and fixed before merging the pull request.